### DISCUSSION TITLE
部署到阿里云时报错

### DISCUSSION THREAD

**Discussion by OrdChaos** [walinejs/waline#890] [Q&A] [answered]
报错信息：Please provide existed ossBucket under your account when code size is greater than 50M.
求教

**Comment by lizheming** [comment, marked as answer]
阿里云 FC 上传代码有体积限制，50M 以上需要先传到 OSS 里面。如果你只是单纯的 clone 仓库的话应该是没有这么大的，你看下是不是你在本地 `npm install` 了导致体积变大了

**Reply by OrdChaos** [reply]
好的，谢谢

**Comment by akatsukiro** [comment]
我没有运行 `npm install` 。是 `s deploy` 时会自己安装node modules，导致文件过大。而这样做是官方文档中所描述的。所以我不知道到底怎么办好了。希望哪位大佬能解答一下。

**Reply by lizheming** [reply]
https://github.com/walinejs/aliyun-fc-starter/blob/master/s.yml#L14-L17 可以把这几行去了试一下

**Reply by akatsukiro** [reply]
> https://github.com/walinejs/aliyun-fc-starter/blob/master/s.yml#L14-L17 可以把这几行去了试一下

依旧不行。报错不变。

**Reply by lizheming** [reply]
https://waline.js.org/guide/server/aliyun-fc.html 那你参考一下评论区的部署方式试试吧

**Reply by akatsukiro** [reply]
好的
